src/video_core/renderer_opengl/renderer_opengl.cpp

```diff
@@ -150,6 +150,8 @@ void RendererOpenGL::SwapBuffers() {
     {
         auto aggregator = Common::Profiling::GetTimingResultsAggregator();
         aggregator->AddFrame(profiler.GetPreviousFrameResults());
+        if (VideoCore::g_toggle_framelimit_enabled)
+        FrameLimiter(aggregator->GetAggregatedResults().frame_time.avg, 60);
```

Comment:
Use explicit braces for the condition and indent properly.
I also think that it was mentioned before that not all 3DS games run at 60 FPS